src/Generators/staticgenerators.jl

```diff
@@ -0,0 +1,345 @@
+abstract type StaticGenerator <: GraphGenerator end
+
+"""
+    struct Complete <: StaticGenerator
+
+A struct representing a generator for a [complete graph](https://en.wikipedia.org/wiki/Complete_graph)
+with `n` vertices.
+
+### Required Fields
+- `n::Integer`: the number of vertices in the graph.
+"""
+struct Complete{T<:Integer} <: StaticGenerator
+    n::T
+end
```

Comment:
We could rid of the directed versions, right? So maybe we should document the behavior in the directed case.

Reply:
There is not separate struct for directed complete graphs. What's the behavior that's different?